frontend/app/common/constants.config.ts

```diff
@@ -0,0 +1,5 @@
+export const BASE_URL: string =
```

Comment:
mb it should be just "config.js"?

Reply:
I don't know, I extracted these constant from `constants.ts` and I suppose it is constants. Anyway it can be removed in future.